CHANGES.md

```diff
@@ -4,12 +4,27 @@ This file is only intended to serve the developers of this repository and of the
 
 This provides an overview of the releases that have been made using this release pipeline.
 
-## (WORK IN PROGRESS) September 2022: Security releases
+## WORK IN PROGRESS October 2022: First 1.37 release
+
+The [standard MediaWiki upgrade process](https://www.mediawiki.org/wiki/Manual:Upgrading) can be followed for this release.
+
+- [MediaWiki release notes](https://www.mediawiki.org/wiki/Release_notes/1.37)
+- [Wikibase release notes](https://github.com/wikimedia/Wikibase/blob/REL1_37/RELEASE-NOTES-1.37)
+
+The Query service was also updated from `0.3.97` to TODO, but no additional upgrade process is needed.
 
 | Suite Version | MediaWiki release | Date availbile | Run number | Release task  |
 |---------------|----------------------|----------------|------------| --------------|
-| wmde.8 spec(TBA) | 1.36    | Date TBA | Run TBA | [T316707](https://phabricator.wikimedia.org/T316707) |
-| wmde.7 spec(TBA) | 1.35    | Date TBA | Run TBA | [T314881](https://phabricator.wikimedia.org/T314881) |
+| wmde.9 ([spec](https://github.com/wmde/wikibase-release-pipeline/blob/wmde.9/versions/wmde9.env),[metadata](https://github.com/wmde/wikibase-release-pipeline/blob/main/versions/wmde9)) | 1.37.TODO    | TODO | [TODO](https://github.com/wmde/wikibase-release-pipeline/actions/runs/TODO) | [T317890](https://phabricator.wikimedia.org/T317890) |
```

Comment:
Would be good to list both MW and PHP versions, not just MW

Reply:
Done in https://github.com/wmde/wikibase-release-pipeline/pull/357/commits/59cf15219809b34e3e636c43158c70863d75c0f0